src/runner/api/itemRunner.js

```diff
@@ -223,46 +223,46 @@ var itemRunnerFactory = function itemRunnerFactory(providerName, data, options)
          * @fires itemRunner#statechange the provider is reponsible to trigger this event
          * @fires itemRunner#responsechange  the provider is reponsible to trigger this event
          */
-        render: function(elt, options) {
-            var self = this;
-
+        render(elt, newOptions = {}) {
             /**
              * Call back when render is done
              */
-            var renderDone = function renderDone() {
+            const renderDone = () => {
                 //manage pending tasks the first time
                 if (flow.render.done === false) {
                     flow.render.done = true;
 
-                    _.forEach(flow.render.pending, function(pendingTask) {
-                        if (_.isFunction(pendingTask)) {
-                            pendingTask.call(self);
-                        }
-                    });
+                    flow.render.pending
+                        .filter(pendingTask => typeof pendingTask === 'function')
+                        .forEach(pendingTask => pendingTask());
+
                     flow.render.pending = [];
                 }
 
                 /**
                  * The item is rendered
                  * @event itemRunner#render
                  */
-                self.trigger('render');
+                this.trigger('render');
 
                 /**
                  * The item is ready.
                  * Alias of {@link itemRunner#render}
                  * @event itemRunner#ready
                  */
-                self.trigger('ready');
+                this.trigger('ready');
             };
 
-            options = _.defaults(options || {}, { state: {} });
+            options = Object.assign(options, newOptions);
```

Comment:
If it was `options || {}` I think better to leave it because `Object.assign(null)` will throw error
```suggestion
            options = Object.assign(options || {}, newOptions);
```